uint16 x
uint16 y
float32 radius
---
float32 nx
float32 ny
float32 nz